native/core/src/events/calendar.rs

```diff
@@ -22,34 +22,18 @@ pub enum EventLocation {
         #[serde(default, skip_serializing_if = "Option::is_none")]
         name: Option<String>,
 
-        /// further description to this location
-        #[serde(default, skip_serializing_if = "Option::is_none")]
-        description: Option<TextMessageEventContent>,
-
         /// Alternative Icon to show with this location
         #[serde(default, skip_serializing_if = "Option::is_none")]
         icon: Option<Icon>,
 
         /// A `geo:` URI [RFC5870] for the location.
         #[serde(default, skip_serializing_if = "Option::is_none")]
         coordinates: Option<String>,
-
-        /// further Link
-        #[serde(default, skip_serializing_if = "Option::is_none")]
-        uri: Option<String>,
```

Comment:
this could be a link to an external website of the location or a link to google maps... even physical locations have virtual representation now